src/Hedgehog/Lazy.fs

```diff
@@ -0,0 +1,24 @@
+﻿// Workaround for a fable issue: https://github.com/fable-compiler/Fable/issues/2069
+#if FABLE_COMPILER
+module Hedgehog.Lazy
+#else
+[<RequireQualifiedAccess>]
+module internal Hedgehog.Lazy
+#endif
+
+let func (f: unit -> 'a) = Lazy<'a>(valueFactory = fun () -> f ())
```

Comment:
can `fun () -> f()` be reduced to just `f`?

Reply:
Unfortunately not because the type of `valueFactory` is `Func<'a>` (i.e. C#/.NET's function type) instead of `unit -> 'a` (i.e. F#'s function type).